projects/libs/flex-layout/package.json

```diff
@@ -29,7 +29,7 @@
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@angular/cdk": "^13.0.0",
+    "@angular/cdk": "^14.0.0",
```

Comment:
Pardon the repeated comment, I wanted to point out the specific line this was referring to

> @CaerusKaru for this and future updates, would it be possible to use peer deps similar to how Angular Material does, which includes the next major version as well, to ease the update process. e.g. `^14.0.0 || ^15.0.0` (https://github.com/angular/components/blob/5816a12dfa816e3f340627fdf561658de0e8c970/packages.bzl#L4) It's not usually the case that the next major Angular or CDK versions break this library, so it's likely safe to do and would improve the DX for this interim period between angular and flex layout releases.